src/commands/command-executor.js

```diff
@@ -388,6 +392,9 @@ class CommandExecutor {
                 command.data = commandInstance.pack(command.data);
             }
         }
+        if (!command.priority) {
+            command.priority = DEFAULT_COMMAND_PRIORITY;
+        }
```

Comment:
When command.priority is set to HIGHEST (0) this will evaluate to true and write DEFAULT_COMMAND_PRIORITY value to db instead